projects/vdk-plugins/vdk-jupyter/getting-started.ipynb

```diff
@@ -0,0 +1,292 @@
+    "\n",
+    "<h4>Data Job directory can contain any files, however, there are some files that are treated in a specific way.</h4>\n",
+    "\n",
+    " \n",
+    "<details>\n",
+    "        <summary><strong style=\"color: Tomato;\">Click to see them!</strong></summary>\n",
+    "        <ul>\n",
+    "            <li><strong style=\"color: blue;\">SQL files (.sql)</strong> - called SQL steps - are directly executed as queries against a database.</li>\n",
+    "            <li><strong style=\"color: green;\">Notebook files (.ipynb)</strong> - can contain labelled VDK cells (called notebook steps) which are executed as steps in a VDK Data Job;</li>\n",
+    "            <li><strong style=\"color: red;\">Python files (.py)</strong> - called Python steps - are python scripts that define a run function that takes as an argument the job_input object.</li>\n",
+    "            <li><strong style=\"color: purple;\">config.ini</strong> is needed in order to schedule a job.</li>\n",
+    "            <li><strong style=\"color: brown;\">requirements.txt</strong> is needed when your Python or Notebook steps use external python libraries.</li>\n",
+    "        </ul>\n",
+    "        <img src=\"./images/data-job-directory.png\" width=\"1000\" length=\"546\" alt=\"Data Job Directory\">\n",
+    "</details>\n",
+    "\n",
+    "\n",
+    "\n",
+    "   <p>VDK supports having many Python, SQL and Notebook steps in a single Data Job. These steps are carried out in an ascending alphabetical sequence based on the file names, and Notebook steps are run from the beginning to the end of the file. For an effective execution order while retaining informative file names, it's beneficial to prefix your file names with numbers.</p>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eb565197",
+   "metadata": {},
+   "source": [
+    "<h2>Step 0: Explore VDK's Functionalities</h2>\n",
+    "    <p>You can execute any shell command if you prefix it with \"!\". \"!vdk --help\" gives you good knowledge on VDK's capabilities.</p>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c7b972ea",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!vdk --help"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0ad33190-232c-48b7-a62b-4d881a0a9b1c",
+   "metadata": {},
+   "source": [
+    "<p>These commands are also readily accessible in the VDK menu located on the top toolbar. \n",
+    "This guide will delve into each of these options for a comprehensive understanding.\n",
+    "We'll also provide you with information on utilizing VDK within Notebooks.</p>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3e038188",
+   "metadata": {},
+   "source": [
+    "<h2>Step 1 Login </h2>\n",
```

Comment:
Radost suggested that we should add something like the workflow expected almost as a table of content 

E.g Login -> Create 
                 -> Download. -> Develop -> Run -> Deploy -> ...
                 
And people can click on it and go to that section